### PR TITLE
pile 0.9.2 (new cask)

### PR DESCRIPTION
**Pull request by pavelzw**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/p/pile.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      verified: "github.com/UdaraJay/Pile/"
+  name "Pile"
+  desc "Desktop app for digital journaling"
+  homepage "https://udara.io/pile/"
+
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
+
```
Comment (bevanjkay) [inline Casks/p/pile.rb:18]: The livecheck block is only required if upstream uses pre-releases. The default `:git` strategy is inferred automatically from the `url` and is preferred.

File: Casks/p/pile.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  depends_on macos: ">= :high_sierra"
+
+  app "Pile.app"
+
+  zap trash: [
+    "~/Library/Preferences/pile.un.ms.plist",
+  ]
```
Comment (bevanjkay) [inline Casks/p/pile.rb:25]:
```suggestion
  zap trash: "~/Library/Preferences/pile.un.ms.plist"
```

**Review by bevanjkay — CHANGES_REQUESTED** [review]
Does the app have an auto-update and install mechanism?
It looks like it may have Electron Builder functionality.
If so it should have `auto_updates true`

**Comment by pavelzw** [comment]
> Does the app have an auto-update and install mechanism?

For now, they don't. https://github.com/UdaraJay/Pile/issues/40#issuecomment-1873501788

**Review by bevanjkay — APPROVED** [review]

**Comment by bevanjkay** [comment]
Thank you @pavelzw 🚀

**Pull request merged**